### PR TITLE
Adds sponsors information

### PR DESCRIPTION
**Pull request by maxcell**
We got info at the top!

image

And info at the bottom!
image

### CODE REVIEW

File: app/assets/stylesheets/sponsors/index.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@import "../colors";
+@import "../layouts";
+
+
```
Comment (stevenpetryk) [inline app/assets/stylesheets/sponsors/index.scss]: Could you get rid of this extra `\n`?

Comment (stevenpetryk) [inline app/assets/stylesheets/sponsors/index.scss]: Could you get rid of this extra `\n`?

File: app/views/application/home.html.erb [hunk 2]
```diff
@@ -7,6 +7,8 @@
   </div>
 
   <a class="button" href="#">Apply</a>
+  <br />
+  <a class="button" href="#">Sponsor us</a>
```
Comment (stevenpetryk) [inline app/views/application/home.html.erb]: 🚨🚨🚨 **`<br />` POLICE, FIRST WARNING** 🚨🚨🚨

PLEASE COME OUT WITH YOUR HANDS UP. THEN USE THOSE HANDS TO USE THE CSS `display: block` ATTRIBUTE TO MAKE THESE BREAK ONTO THEIR OWN LINES.

ANY FURTHER USE OF `<br />` WILL BE REPORTED TO STATE OFFICIALS AND BACKGROUND CHECKING SERVICES.

Comment (maxcell) [inline app/views/application/home.html.erb]: BB `.button` has the display block on it but I just didn't want to add paddings :(

Comment (stevenpetryk) [inline app/views/application/home.html.erb]: Oh okay, margins should work then!

File: app/assets/stylesheets/sponsors/index.scss (hunk 1 above)
Comment (maxcell) [inline app/assets/stylesheets/sponsors/index.scss]: Got rid of that in the new commit :+1:

File: app/views/application/home.html.erb (hunk 2 above)
Comment (maxcell) [inline app/views/application/home.html.erb]: ❤️ Margins was perfect.

**Comment by maxcell** [comment]
Worked out really well! Thank you for your great feedback!
image

**Pull request merged**